HelpCentre/QA/HelpCentre: Agent Wrap-up.md

```diff
@@ -0,0 +1,501 @@
+  buttons(
+    CallMeBackConfirm: "@button_labels[0]",
+    GotoMainMenu: "@button_labels[1]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## Choose number to call
+
+```stack
+card ChooseNumberToCall, then: ChooseNumberToCallError do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_agent_call_back_number_confirmation"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  choose_number_msg = page.body.body.text.value.message
+  button_labels = map(page.body.body.text.value.buttons, & &1.value.title)
+
+  buttons(
+    UseThisNumber: "@button_labels[0]",
+    UseAnotherNumber: "@button_labels[1]",
+    GotoMainMenu: "@button_labels[2]"
+  ) do
+    text("@choose_number_msg")
+  end
+end
+
+card UseThisNumber, then: BotToAgentHandoverWait do
+  log("UseThisNumber")
+end
+
+card ChooseNumberToCallError, then: ChooseNumberToCallError do
+  buttons(
+    UseThisNumber: "@button_labels[0]",
+    UseAnotherNumber: "@button_labels[1]",
+    GotoMainMenu: "@button_labels[2]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## Use another number
+
+```stack
+card UseAnotherNumber, then: ValidateAlternateNumber do
+  # HC: Scheduled - Callback follow up
+  cancel_scheduled_stacks("36ec8712-99c2-453c-b6cb-fbe9f7cf4bae")
+  schedule_stack("36ec8712-99c2-453c-b6cb-fbe9f7cf4bae", in: 900)
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_agent_call_back_number_update"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  update_number_msg = page.body.body.text.value.message
+
+  cancel_scheduled_stacks("36ec8712-99c2-453c-b6cb-fbe9f7cf4bae")
+
+  log(
+    "Cancelling previously scheduled stack `HC: Scheduled - Callback follow up - 36ec8712-99c2-453c-b6cb-fbe9f7cf4bae`"
+  )
+
+  schedule_stack("36ec8712-99c2-453c-b6cb-fbe9f7cf4bae", in: 900)
+
+  log(
+    "Scheduled stack `HC: Scheduled - Callback follow up - 36ec8712-99c2-453c-b6cb-fbe9f7cf4bae` to run in 2 minutes"
+  )
+
+  preferred_callback_number =
+    ask("@update_number_msg")
+end
+
+card ValidateAlternateNumber, then: ConfirmSaveAlternateNumber do
+  text("TODO: ValidateAlternateNumber")
```

Comment:
I didn't want to use it at this stage, as it will confuse testing and users if they input something valid, that gets detected as invalid.  Plan is to come back to this once I have resolved the ticket of discussing the issue with Turn. You OK with that?